### ISSUE TITLE
GSet.balance doesn't parse any more

### ISSUE THREAD

**Issue by simonhorlick** [HigherOrderCO/Kind#147]
It looks like the recent changes to `else` have cause `GSet.balance` to not compile any more:

```
$ kind GSet/balance.kind
On 'GSet/balance.kind':
Expected 'else', found end of file.
  65 |           }
  66 |     }
  67 |   else // neither subtrees are too heavy, no balancing is needed 
```

**Comment by elmattic** [comment]
Maybe using `()` around a `case`/`switch` expression to disambiguate parsing?

**Comment by simonhorlick** [comment]
Fixed in 1.0.14. Thanks!
